File: core-java-practice/gcr-codebase/AddTwoNumbers.java
import java.util.Scanner;

public class AddTwoNumbers {
    public static void main(String[] args) {
        //read input
        Scanner sc=new Scanner(System.in);

        int number1=sc.nextInt(); //take number 1
        int number2=sc.nextInt(); //take number 2

        //variable to store the sum of numbers
        int c=number1+number2;
        
        //printing the sum 
        System.out.println("The sum of the two numbers is: " + c);
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/AddTwoNumbers.cs
using System;

class AddTwoNumbers {
    static void Main(string[] args) {
        //read input
        int number1 = int.Parse(Console.ReadLine()); //take number 1
        int number2 = int.Parse(Console.ReadLine()); //take number 2

        //variable to store the sum of numbers
        int c = number1 + number2;
        
        //printing the sum 
        Console.WriteLine("The sum of the two numbers is: " + c);
    }
}